src/client/pages/OfferNew/utils.ts

```diff
@@ -178,8 +179,14 @@ export const hasCurrentInsurer = (quote: OfferQuote): boolean =>
   Boolean(quote.currentInsurer)
 
 export const isStudent = (details: QuoteDetails) => {
```

Comment:
This is used for all markets since we have student quotes in Denmark as well (not Norway - there we have youth quotes instead 😅). 

But the way we can detect if it's a student quote differs unfortunately. For Sweden we have to check the `type` on the quote but in Denmark there is an `isStudent` property on the quote. So that's checked in the second `if` statement of this function.

Reply:
It is used across all markets? But doesn't this function only work for Swedish quotes then?

Reply:
Thank you Gustav for explaining it more clearly 😄